app/src/config/config-overrides-generated.ts

```diff
@@ -1,6 +1,5 @@
 // Auto-generated file, do not modify directly
```

Comment:
Can you help add this to the git ignore? I've added this, but it doesn't stay ignored just like environment.js under the api/lambda folder sometimes shows up not ignored?